pkg/infra/log/slog.go

```diff
@@ -0,0 +1,60 @@
+package log
+
+import (
+	"context"
+	"log/slog"
+)
+
+var _ slog.Handler = &SLogHandler{}
+
+type SLogHandler struct {
+	Logger
+}
+
+// NewSLogHandler returns a SLogHandler with the given level.
+func NewSLogHandler(logger Logger) *SLogHandler {
```

Comment:
Maybe in a child package instead, slogadapter, with `New`?

Reply:
updated in [de75497](https://github.com/grafana/grafana/pull/84680/commits/de75497dc93d208bd0ef0d7e26b689aeffb71a50)